### DISCUSSION TITLE
ShellHost.exe -System Error

### DISCUSSION THREAD

**Discussion by Kawika498** [simeononsecurity/Windows-Optimize-Harden-Debloat#102] [Q&A]
I did a fresh install of Windows on this Desktop I am hardening After running the script and reseating the device when I log into my Admin user the desktop keeps crashing and a prompt saying that the system detected an overrun of stack-based buffer in this application. Are there any idea on what might be causing this I have done 3 fresh installs just to make sure it wasn't an application that was causing the issue

**Comment by simeononsecurity** [comment]
https://github.com/simeononsecurity/Windows-Optimize-Harden-Debloat/issues/103
